tests/main/snap-pack/task.yaml

```diff
@@ -0,0 +1,27 @@
+summary: check that snap pack creates squashfs files
+description: |
+    Verify that snap pack creates squashfs files or fails in a predictable
+    manner.
+
+systems: [-ubuntu-core-*]
```

Comment:
I think it can run on core as well, no?

Reply:
I agree, it does seem like it could run on core too.
If it can't I think a comment explaining why it can't run on core is warranted